mindsdb/integrations/handlers/llama_index_handler/llama_index_handler.py

```diff
@@ -269,31 +256,22 @@ def predict(
 
     def _get_service_context(self):
         args = self.model_storage.json_get("args")
-        engine_storage = self.engine_storage
-
-        key = "OPENAI_API_KEY"
-        openai_api_key = get_api_key(
-            key, "llama_index", args["using"], engine_storage, strict=False
-        )
-        if openai_api_key is None:
-            openai_api_key = get_api_key(
-                key.lower(), "llama_index", args["using"], engine_storage, strict=True
-            )
-
+        openai_api_key = self._get_llama_index_api_key(args["using"])
         openai.api_key = openai_api_key  # TODO: shouldn't have to do this! bug?
-        llm_kwargs = {"openai_api_key": openai_api_key}
+        llm_kwargs = {
+            "openai_api_key": openai_api_key,
+            "model": "gpt-3.5-turbo-instruct",
```

Comment:
Let's switch to `gpt-3.5-turbo`, following [OpenAI's official recommendation](https://platform.openai.com/docs/models/gpt-3-5). Also, please comment this line to indicate we should remove this specific parameter enforcing once the `langchain` dependency is updated.

Reply:
We need instruct models for this handler like the `text-davinci-003`, passing a chat model throws error from openai. Something like `Received v1 endpoints chat model` .

Reply:
currently passing a chat model like `gpt-3.5-turbo` will fail due to OpenAI's migration which gives below error. So I have added a `v1/completion` version of `gpt-3.5-turbo` which is `gpt-3.5-turbo-instruct`. 
<img width="1415" alt="Screenshot 2024-01-16 at 9 37 38 PM" src="https://github.com/mindsdb/mindsdb/assets/94780581/8dd1ca57-555d-456b-8320-ce8071e06cac">